project/Build.scala

```diff
@@ -2056,7 +2056,7 @@ object Build {
               Some(ExpectedSizes(
                   fastLink = 425000 to 426000,
                   fullLink = 282000 to 283000,
-                  fastLinkGz = 61000 to 62000,
+                  fastLinkGz = 60000 to 61000,
```

Comment:
Heh, nice. Do you know what that is?

Reply:
I'm not exactly sure. Apparently the inlining of `val`s can be a little bit better in some cases. This is the diff: https://gist.github.com/sjrd/28e5929fa16e0e030a3384bfad1fe59b